src/brainstem-params.coffee

```diff
@@ -0,0 +1,8 @@
+class BrainstemParams
```

Comment:
I think I'm just totally unclear about why this is needed. What would break if someone just used a POJO instead of `BrainstemParams`? Why can't we just support people adding in their filters, etc. to a POJO?

Reply:
Yeah its due to the implementation of `wrapObjects` in `Utils.coffee`. And the way `wrapObjects` is used to wrap `includes` and generate a stack to request each layer of associations.